openadmet/models/features/gat_featurizer.py

```diff
@@ -0,0 +1,175 @@
+from typing import Optional, List, Any
+import torch
+from torch_geometric.data import Data
+from torch_geometric.loader import DataLoader
+from loguru import logger
+
+try:
+    from rdkit import Chem
+except ImportError:
+
+    logger.warning(
+        "RDKit library not found. GraphFeaturizer will not be functional. "
+        "Please install RDKit (e.g., conda install -c conda-forge rdkit)."
+    )
+    Chem = None # Set to None to check its availability later
+
+# Assuming feature_base.py is in the same directory or accessible via PYTHONPATH
```

Comment:
Is there a specific reason this should be necessary? Registering and also importing in `openadmet/models/registries.py` should do the job.